### ISSUE TITLE
Seeking Aid mission nearly impossible

### ISSUE THREAD

**Issue by the2nake** [endless-sky/endless-sky#7879]
### Problem Description

Oathkeeper ships keep destroying the Smuggler ships before I can board them. I'm using a Falcon with the Caldera afterburner, but I'm still nowhere near as fast as is needed to board smuggler ships before they're destroyed by particle cannons.

### Related Issue Links

None.

### Desired Solution

Make Smuggler ships mission targets, and make every faction neutral to them (similar to how bounty pirate ships are neutral to every faction).

### Alternative Approaches

Smuggler ships could receive much stronger hulls than usual (this wouldn't affect gameplay because the one you board self-destructs).

**Comment by Quantumshark** [comment]
It feels odd to me that smugglers are considered to be friendly toward the player, but are still attacked by Navy ships.

**Comment by Hurleveur** [comment]
They should probably be ignored or they're not really smugglers at all, just pirates.
What I did is just go deep in pirate territory where there are almost no navy

**Comment by warp-core** [comment]
The Navy and others will no longer attack the smugglers, but they may still be difficult to identify, disable, and board successfully for other reasons, especially for someone who doesn't already know what ships to look for, and where.

**Comment by roadrunner56** [comment]
I agree with warp on this one. The last time I tried this mission, I had to circle through every ship in the system until I finally found a smuggler, and fly over to that smuggler before they jumped out of the system. This was extremely tedious and I didn't like it. Making the smuggler a target would make things much more reasonable.
